### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jsmolar**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/test-case.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+**Documentation**
+Link any relevant product documentation.
+
+For e.g. Kuadrant Getting started doc: https://github.com/Kuadrant/authorino/blob/main/docs/getting-started.md
+
+**Additional context**
+Add any other context relevant to the feature request here (or skip this section if you are happy with the above).
+
+**Test cases**
+Split test case into sub-tasks if needed. It should help with trackng progress and navigating more complex tasks.
```
Comment (guicassolato) [inline .github/ISSUE_TEMPLATE/test-case.md:22]:
```suggestion
Split test case into sub-tasks if needed. It should help with tracking progress and navigating more complex tasks.
```

File: .github/ISSUE_TEMPLATE/test-case.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+---
+
+**Summary**
+Provide description and important information.
+
+**Documentation**
+Link any relevant product documentation.
+
+For e.g. Kuadrant Getting started doc: https://github.com/Kuadrant/authorino/blob/main/docs/getting-started.md
```
Comment (guicassolato) [inline .github/ISSUE_TEMPLATE/test-case.md:16]:
```suggestion
E.g. for Authorino Getting started doc: https://github.com/Kuadrant/authorino/blob/main/docs/getting-started.md
```

File: .github/ISSUE_TEMPLATE/test-case.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+name: New Test Case
+about: New test case definition
+title: ''
+labels: ''
```
Comment (pehala) [inline .github/ISSUE_TEMPLATE/test-case.md:5]: Can you straight up add a "test-case" label there?

**Review by pehala — COMMENTED** [review]
Looks good, only thing I would do is to squash the commit when you are done

**Review by pehala — DISMISSED** [review]

**Review by pehala — APPROVED** [review]

**Review by JaurbanRH — APPROVED** [review]

**Pull request merged**